src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/CustomCaConfigurationTest.java

```diff
@@ -110,7 +112,8 @@ void setup(ExtensionContext context) throws DeviceConfigurationException {
         kernel = new Kernel();
         kernel.getContext().put(GroupManager.class, groupManager);
         kernel.getContext().put(SecurityService.class, securityServiceMock);
-        kernel.getContext().put(CertificateExpiryMonitor.class, certExpiryMonitor);
+        kernel.getContext().put(CertificateExpiryMonitor.class, certExpiryMonitorMock);
+        kernel.getContext().put(CISShadowMonitor.class, cisShadowMonitorMock);
```

Comment:
Mocking the cisShadowMonitor given it was always getting triggered trying to subscribe during this tests and always fails (we don't care about the cisShadowMonitor) in the context of this test.